python/common/org/python/types/FrozenSet.java

```diff
@@ -124,4 +124,11 @@ public FrozenSet(org.python.Object[] args, java.util.Map<java.lang.String, org.p
     public org.python.Object __not_contains__(org.python.Object other) {
         return new org.python.types.Bool(!this.value.contains(other));
     }
+
+    @org.python.Method(
+            __doc__ = "Return a shallow copy of a FrozenSet."
+    )
+    public org.python.Object copy() {
+        return this;
```

Comment:
@freakboy3742 It's an immutable Set. So, making a copy of the original leads to memory waste. That's why a reference is returned.

Refer : [http://stackoverflow.com/questions/37100944/i-think-immutable-types-like-frozenset-and-tuple-not-actually-copied](http://stackoverflow.com/questions/37100944/i-think-immutable-types-like-frozenset-and-tuple-not-actually-copied-what-is-th)